MobileWallet/Screens/Send/AddRecipient/AddRecipientViewController.swift

```diff
@@ -398,7 +398,7 @@ class AddRecipientViewController: UIViewController, UITextFieldDelegate, Contact
 
         let pasteboardString: String? = UIPasteboard.general.string
 
-        if let text = pasteboardString {
+        if let text = pasteboardString?.emojiString {
```

Comment:
Will the line below still work if you extract only emojis from the clipboard?
```swift
let pubKeyFromDeeplink = try PublicKey(any: text)
```

Reply:
yes